src/Command/FetchConferencesCommand.php

```diff
@@ -43,55 +45,98 @@ protected function configure()
         $this->setDescription('Fetch conferences from Fetcher Classes');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    /**
+     * @param InputInterface $input
+     * @param ConsoleOutputInterface $output
+     * @return int
+     */
```

Comment:
Pourquoi ? Le prototype de ta méthode est déjà bien typé.

Reply:
Not exactly, I use `ConsoleOutputInterface` in PHPDoc, which allows completion for `section` method